apps/browser-extension-wallet/src/hooks/useWalletManager.ts

```diff
@@ -528,6 +574,78 @@ export const useWalletManager = (): UseWalletManager => {
     [getCurrentChainId]
   );
 
+  const createSharedWallet = async ({
+    mnemonic,
+    name,
+    password,
+    chainId = getCurrentChainId(),
+    publicKeys,
+    ownSignerWalletId
+  }: CreateSharedWalletParams): Promise<Wallet.CardanoWallet> => {
+    const accountIndex = 0;
+    const passphrase = Buffer.from(password, 'utf8');
+    const keyAgent = await Wallet.KeyManagement.InMemoryKeyAgent.fromBip39MnemonicWords(
+      {
+        chainId,
+        getPassphrase: async () => passphrase,
+        mnemonicWords: mnemonic,
+        accountIndex,
+        purpose: KeyManagement.KeyPurpose.MULTI_SIG
+      },
+      {
+        bip32Ed25519: Wallet.bip32Ed25519,
+        logger
+      }
+    );
+
+    const multiSigParticipants = publicKeys.sort();
```

Comment:
The script has to also include own key, or else whoever creates the wallet won't be part of it.
```suggestion
    const multiSigParticipants = [ownXPub, ...publicKeys].sort();
```

Reply:
I passed it [here](https://github.com/input-output-hk/lace/blob/aaf64f5c7cefe93359ba01e3cbb4a8d59267959b/apps/browser-extension-wallet/src/views/browser-view/features/shared-wallet/SharedWallet.tsx#L55-L56) already.